### ISSUE TITLE
Question regarding related paper

### ISSUE THREAD

**Issue by seungwooYoo** [NVIDIA/apex#52]
Hi thanks for sharing nice library :)

I have a question regarding this library. Is the library is related to ICLR 18 paper from NVIDIA (Mixed precision training - https://openreview.net/forum?id=r1gs9JgRZ) ? If not then any other references to the library ?

**Comment by mcarilli** [comment]
Yes, the Apex tools are meant to help automate the mixed precision recipe described in that paper.  Note that in our case, we did find loss scaling to be necessary to achieve mixed precision convergence with Resnet50, see https://github.com/NVIDIA/apex/issues/43#issuecomment-421924392.

**Comment by mcarilli** [comment]
These talks provide some additional useful explanations of mixed precision training:
https://github.com/NVIDIA/apex/issues/43#issuecomment-422657924
